packages/datatrak-web/src/components/TaskMetric.tsx

```diff
@@ -0,0 +1,52 @@
+import { SpinningLoader } from '@tupaia/ui-components';
+import React from 'react';
+import styled from 'styled-components';
+
+const MetricWrapper = styled.div`
+  display: flex;
+  border: 1px solid;
+  border-radius: 3px;
+  margin-inline: 0.5rem;
+  margin-block-end: auto;
+  ${({ theme }) => theme.breakpoints.down('xs')} {
+    width: inherit;
+    margin-block-start: 0.5rem;
+    margin-inline: 0;
+  }
+`;
+
+const MetricNumber = styled.div`
+  font-size: 14px;
+  line-height: 1.75;
+  padding: 0.5em 1.75em;
```

Comment:
```suggestion
  padding: 0.5em 1.75rem;
```

We use `rem` as opposed `em` because `em` will sometimes come with unexpected results depending on the parent's font size